requirements.txt

```diff
@@ -39,4 +39,6 @@ thrift-sasl==0.3.0
 unicodecsv==0.14.1
 unidecode==1.0.22
 contextlib2==0.5.5
-
+croniter==0.3.24
```

Comment:
@mistercrunch What is the project's preference on pinning versions? Should we keep it fixed or leave it as `>=` ?